mods/ra/rules/defaults.yaml

```diff
@@ -831,6 +831,7 @@
 		CrushClasses: mine
 		DetonateClasses: mine
 		AvoidFriendly: false
+		BlockFriendly: true
```

Comment:
I thought it would be useful to have booleans related to mine behaviour in one place...